docs/agents/sematext-agent/kubernetes/helm.md

```diff
@@ -0,0 +1,108 @@
+## Introduction
+
+This chart installs the Sematext Agent to all nodes in your cluster via a `DaemonSet` resource.
+
+## Prerequisites
+
+- Kubernetes 1.13+
+- You need to create [an Infra App in Sematext Cloud](https://apps.sematext.com/ui/monitoring-create/app/infra) to get your infra token.
+
+## Installation
+
+To install the chart for monitoring and shipping logs run the following command:
+
+```bash
+$ helm repo add sematext https://cdn.sematext.com/helm-charts/
+$ helm install st-agent \
+    --set infraToken=YOUR_INFRA_TOKEN \
+    --set region=US \
+    sematext/sematext-agent
+```
+
+To install the chart for both logs and monitoring run the following command:
+
+```bash
+$ helm repo add sematext https://cdn.sematext.com/helm-charts/
+$ helm install st-agent \
+    --set logsToken=YOUR_LOGS_TOKEN \
+    --set infraToken=YOUR_INFRA_TOKEN \
+    --set region=US \
+    sematext/sematext-agent
+```
+
+To provide your infra token as a kubernetes secret instead, create a secret with `infra-token` as a key, and provide its name to the install command:
```

Comment:
Is there an advantage to doing it this way?  If yes, maybe we can say a sentence or two about that.

Reply:
Added